src/pages/api/health.ts

```diff
@@ -0,0 +1,5 @@
+import { NextApiRequest, NextApiResponse } from "next";
+
+export default (req: NextApiRequest, res: NextApiResponse): void => {
+  res.status(200).json({ status: "Ship shape and Bristol fashion" });
+};
```

Comment:
So much this.  I wonder what happens if you put an emoji in there... 🏴‍☠️⚓️🚢

Reply:
_I WAS THINKING THE SAME_

Reply:
But I didn't want to push it... 😛